shared/src/test/diff/mlscript/TupleArray.mls

```diff
@@ -319,14 +319,14 @@ hg1 ((5,5,5))
 
 def ta1: Array[int] | (int, bool)
 def test: (string, 1) & { _1: "hello" }
-//│ ta1: Array[bool | int]
+//│ ta1: Array[Array[bool | int] | int]
```

Comment:
You should always have a quick look over what diff-test changes your modifications had. Here you can clearly see that something went wrong.